internal/witcli/witcli_test.go

```diff
@@ -0,0 +1,69 @@
+package witcli_test
```

Comment:
This module doesn't use test packages, is there a specific reason?

Reply:
Oversight on my part, fixed!